docs/book/src/developer/providers/contracts/infra-cluster.md

```diff
@@ -335,6 +335,12 @@ the implication of this choice which are described both in the document above an
 
 </aside>
 
+### InfraCluster: Pausing
```

Comment:
Please link this rule from the table up at L44 on this page (you can add this as a new line at the end)

Reply:
I've added at the end, not sure if it should be underneath the other `InfraCluster: <foo>` lines though?